src/components/Cards/PlaylistCard/PlaylistCard.js

```diff
@@ -0,0 +1,47 @@
+import "./PlaylistCard.css";
+import { usePlaylist } from "../../../context/playlistContext";
+import { useNavigate } from "react-router-dom";
+const PlaylistCard = ({ playlist }) => {
+  const navigate = useNavigate();
+  // console.log(playlist);
+  const { removePlaylist } = usePlaylist();
+  console.log(playlist.videos[0].img_src);
+  return (
+    <div className="bord-3-purple playlist-card-container flex-col">
+      <div className="playlist-thumbnail-container bord-3-green">
+        {playlist.videos.length === 0 ? (
+          <div className="playlist-thumbnail-section">
+            <h3>Empty playlist</h3>
+          </div>
+        ) : (
+          <img
+            className="img-resp"
+            src={playlist.videos[0].img_src}
+            alt="thumbnail-img"
```

Comment:
the alt attribute adds image/picture etc by default when read by a screen reader.
suggestion: video thumbnail instead